pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -878,13 +878,19 @@ func (p *PriorityQueue) Update(logger klog.Logger, oldPod, newPod *v1.Pod) {
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	// When unscheduled Pods are updated, we check with QueueingHint
+	// whether the update may make the pods schedulable.
+	// Plugins have to implement a QueueingHint for Pod/Update event
+	// if the rejection from them could be resolved by updating unscheduled Pods itself.
+	events := framework.PodSchedulingPropertiesChange(newPod, oldPod, true)
```

Comment:
Will L899 below actually haapen?  @sanposhiho https://github.com/kubernetes/kubernetes/blob/b8163a856115230464b815ed3acd28f8c76631c8/pkg/scheduler/backend/queue/scheduling_queue.go#L899

If this could happen, then we should check if oldPod is nil in `PodSchedulingPropertiesChange`, but I think it will never happen right? https://github.com/kubernetes/kubernetes/blob/e456fbfaa648874e0b10c27c5f56c4f3e8c71d03/pkg/scheduler/eventhandlers.go#L157

Reply:
> Will L899 below actually haapen?

Yes. Actually, it should always be `oldPod != nil` basically, right?